qiita_db/util.py

```diff
@@ -1098,3 +1099,60 @@ def infer_status(statuses):
     # If there are no statuses, or any of the previous ones have been found
     # then the inferred status is 'sandbox'
     return 'sandbox'
+
+
+def add_message(message, users):
+    """Adds a message to the messages table, attaching it to given users
+
+    Parameters
+    ----------
+    message : str
+        Message to add
+    users : list of User objects
+        Users to connect the message to
+    """
+    with TRN:
+        sql = """INSERT INTO qiita.message (message) VALUES (%s)
+                 RETURNING message_id"""
+        TRN.add(sql, [message])
+        msg_id = TRN.execute_fetchlast()
+        sql = """INSERT INTO qiita.message_user (email, message_id)
+                 VALUES (%s, %s)"""
+        sql_args = [[user.id, msg_id] for user in users]
+        TRN.add(sql, sql_args, many=True)
+        TRN.execute()
+
+
+def add_system_message(message, expires):
+    """Adds a system message to the messages table, attaching it to asl users
+
+    Parameters
+    ----------
+    message : str
+        Message to add
+    expires : datetime object
+        Expiration for the message
+    """
+    with TRN:
+        sql = """INSERT INTO qiita.message (message, expiration)
+                 VALUES (%s, %s)
+                 RETURNING message_id"""
+        TRN.add(sql, [message, expires])
+        msg_id = TRN.execute_fetchlast()
+        sql = """INSERT INTO qiita.message_user (email, message_id)
+                 SELECT email, %s FROM qiita.qiita_user"""
+        TRN.add(sql, [msg_id])
+        TRN.execute()
+
+
+def clear_system_messages():
+    with TRN:
+        sql = """SELECT message_id FROM qiita.message WHERE expiration < %s"""
```

Comment:
You don't need triple quotes here as it fits in a single line. It applies to the other 2 SQL commands below.